tests/test_meetings.py

```diff
@@ -31,55 +33,72 @@ def test_meetings_landing(self, meetings_page):
         assert meetings_page.nrao_logo.present()
         assert meetings_page.spsp_logo.present()
 
-    def test_filtering(self, meetings_page):
+    def test_filtering(self, meetings_page, driver):
+        search_bar = driver.find_element_by_css_selector('div[data-test-meetings-list-search]')
+        driver.execute_script('arguments[0].scrollIntoView();', search_bar)
+
         default_top_result = meetings_page.top_meeting_link.text
         meetings_page.filter_input.clear()
-        meetings_page.filter_input.send_keys('ea')
+        meetings_page.filter_input.send_keys('f')
+        time.sleep(.5)
         filtered_top_result = meetings_page.top_meeting_link.text
         assert default_top_result != filtered_top_result
 
-    def test_carets(self, meetings_page):
+    def test_carets(self, meetings_page, driver):
+        search_bar = driver.find_element_by_css_selector('div[data-test-meetings-list-search]')
+        driver.execute_script('arguments[0].scrollIntoView();', search_bar)
+
         default_top_result = meetings_page.top_meeting_link.text
-        meetings_page.sort_caret_name_desc.click()
+        meetings_page.sort_caret_name_asc.click()
         sorted_top_result = meetings_page.top_meeting_link.text
         assert default_top_result != sorted_top_result
 
     @markers.core_functionality
     def test_meetings_list(self, meetings_page, driver):
+        search_bar = driver.find_element_by_css_selector('div[data-test-meetings-list-search]')
+        driver.execute_script('arguments[0].scrollIntoView();', search_bar)
+
         meeting_name = meetings_page.top_meeting_link.text
         meetings_page.top_meeting_link.click()
         meeting_detail = MeetingDetailPage(driver, verify=True)
         assert meeting_name == meeting_detail.meeting_title.text.strip()
 
 
-@pytest.mark.skipif(settings.STAGE2, reason='No meetings on staging2')
-@pytest.mark.skipif(settings.TEST, reason='Only one meeting on test')
+# @pytest.mark.skipif(settings.STAGE2, reason='No meetings on staging2')
+@pytest.mark.skipif(settings.STAGE1, reason='Only one meeting on test')
```

Comment:
```suggestion
@pytest.mark.skipif(settings.STAGE1, reason='Only one meeting on staging1')
```
^ Fixed copy/paste error.

As I was saying earlier in this review, if we set up the test meetings and submissions properly on all servers, we should actually be ok, and might be able to forego "skipif" all together.